### PR TITLE
Imports 2019 03 29

### PR DESCRIPTION
**Pull request by GeoWill**

### CODE REVIEW

File: polling_stations/apps/data_collection/management/commands/import_tameside.py [hunk 1]
```diff
@@ -3,23 +3,89 @@
...
 
     def address_record_to_dict(self, record):
         uprn = record.property_urn.strip().lstrip("0")
-
-        if record.addressline6 == "OL5 0EX":
-            return None
+        rec = super().address_record_to_dict(record)
+        postcode = record.addressline6
+        #     if record.addressline6 == "OL5 0EX":
+        #         return None
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_tameside.py]: if we don't want this, can we just remove it

File: polling_stations/apps/data_collection/management/commands/import_tameside.py [hunk 2]
```diff
@@ -3,23 +3,89 @@
...
+            "10090073764",
+            "200004407864",
+            "200004407867",
+            "200004410055",
+            "200004410061",
+            "200004410550",
+            "200004410656",
+            "200004422675",
+            "200004423681",
+        ] or postcode in ["SK14 1EY", "SK14 4XE", "SK14 4EG", "M34 2WS", "OL67FU"]:
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_tameside.py]:
```suggestion
        ] or postcode in ["SK14 1EY", "SK14 4XE", "SK14 4EG", "M34 2WS", "OL6 7FU"]:
```

OL67FU won't match anything

Comment (GeoWill) [reply polling_stations/apps/data_collection/management/commands/import_tameside.py]: Accepted and fixed up

File: polling_stations/apps/data_collection/management/commands/import_tameside.py (hunk 1 above)
Comment (GeoWill) [reply polling_stations/apps/data_collection/management/commands/import_tameside.py]: :+1:

**Pull request merged**